docs/BR.md

```diff
@@ -1357,50 +1358,42 @@ No stipulation.
 
 ### 4.9.9 On-line revocation/status checking availability
 
-The following SHALL apply for communicating the status of Certificates which include an Authority Information Access extension with an id-ad-ocsp accessMethod.
+The validity interval of an OCSP response is the difference in time between the `thisUpdate` and `nextUpdate` field, inclusive. For purposes of computing differences, a difference of 3,600 seconds shall be equal to one hour, and a difference of 86,400 seconds shall be equal to one day, ignoring leap-seconds.
 
-OCSP responses MUST conform to RFC6960 and/or RFC5019. OCSP responses MUST either:
+A certificate serial is "assigned" if:
 
-1. Be signed by the CA that issued the Certificates whose revocation status is being checked, or
-2. Be signed by an OCSP Responder whose Certificate is signed by the CA that issued the Certificate whose
-revocation status is being checked.
+- a Certificate or Precertificate with that serial number has been issued by the Issuing CA, using any current or previous key associated with that CA subject; or
+- a Precertificate with that serial number has been issued by a Precertificate Signing Certificate, as defined in [Section 7.1.2.4](#7124-technically-constrained-precertificate-signing-ca-certificate-profile), associated with the Issuing CA.
 
-In the latter case, the OCSP signing Certificate MUST contain an extension of type `id-pkix-ocsp-nocheck`, as
-defined by RFC6960.
+A certificate serial is "unassigned" if it is not "assigned".
 
-### 4.9.10 On-line revocation checking requirements
+The following SHALL apply for communicating the status of Certificates and Precertificates which include an Authority Information Access extension with an id-ad-ocsp accessMethod.
 
-The following SHALL apply for communicating the status of Certificates which include an Authority Information Access extension with an id-ad-ocsp accessMethod.
+Effective 2025-01-15, authoritative OCSP responses for Subscriber Certificates MUST be available (i.e. the responder MUST NOT respond with the "unknown" status) starting no more than 15 minutes after the certificate signing operation occurs.
```

Comment:
> starting no more than 15 minutes after the certificate signing operation occurs.

Could this be modified to focus on the publication of the certificate as opposed to the certificate signing operation? 

1. It is difficult to know precisely when the certificate signing operation occurred from an external auditing standpoint (e.g. backdating the notBefore by a small amount is common).
2. There may be hiccups in the OCSP pipeline which don't affect the initial certificate signing operation. A CA which signs a precertificate, but is unable to push new OCSP for 15 minutes, shouldn't be penalized if the precertificate hasn't been sent to CT yet or relayed to the subscriber.

Proposed text: "starting no more than 15 minutes after the certificate is published or is made available to the subscriber.”

Reply:
Intriguing. I like this idea, but have some hesitation. Two questions:

1. While the publication date can be taken to be "SCT + MMD", not all certificates are published to CT logs, and it's similarly impossible for an external auditor to determine when the certificate is "made available to the subscriber". Does this phrasing truly solve the difficulty, or just move it to a different location?
2. If we were to make this change, how do you think it should interact with the limitation of this requirement to just Subscriber Certificates? If we change this to be within 15 minutes of publication, then it could potentially be applied to CA certificates as well.

Reply:
Do we need additional clarification that this also applies for the moment a precertificate is signed?

Reply:
I'm going to go with 
> ... starting no more than 15 minutes after the Certificate or corresponding Precertificate is published or otherwise made available."